test/stresstest/config.ini

```
@@ -0,0 +1,11 @@
+[vss]
+
+    ip=127.0.0.1
+    port=55555
+    protocol=ws
+    insecure=False
+    subscriberCount=1
+    path=Vehicle.Speed
+    messageNum=1000
+    Debug=1
+    container=True
```

```suggestion
[vss]
ip = 127.0.0.1
port = 55555
protocol = ws
insecure = False
subscriberCount = 1
path = Vehicle.Speed
messageNum = 1000
Debug = 1
container = True
```
`.ini` files usually:

- are unindented
- have no emtpy lines between a section title and its content but one empty line between sections (although you only have one section in this case)
- have spaces around equal signs.
For example: https://docs.python.org/3.8/library/configparser.html?highlight=configparser#quick-start